packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -135,8 +137,13 @@ def test_logger_propagation(logger: logging.Logger):
 def _log_arguments(
     logger_obj: logging.Logger, level: int, func: Callable, *args, **kwargs
```

Comment:
at this point it would make sense to drop the `*args`, but I don't think that is doable. since secrets can still escape via it.

Reply:
I see your point. That is indeed what I suggested in the comment.But to remove *args it requires reviewing all the references to this call.